app/src/main/java/org/oppia/app/story/StoryFragmentScroller.kt

```diff
@@ -0,0 +1,5 @@
+package org.oppia.app.story
+
+interface StoryFragmentScroller {
```

Comment:
Please add KDoc (specifically explaining what this does, not why we have it).

Reply:
Okay

Reply:
More specifically: ``Scrolls smoothly (with animation) to the specified vertical pixel position in [StoryFragment].``

'Handles scrolling' is vague--we can and should be more specific.

Reply:
Done